miio/integrations/humidifier/zhimi/airhumidifier.py

```diff
@@ -108,23 +110,45 @@ def temperature(self) -> Optional[float]:
         return None
 
     @property
+    @sensor("Humidity", unit="%", device_class="humidity")
     def humidity(self) -> int:
         """Current humidity."""
         return self.data["humidity"]
 
     @property
+    @switch(
+        name="Buzzer",
+        icon="mdi:volume-high",
+        setter_name="set_buzzer",
+        device_class="switch",
+        entity_category="config",
+    )
     def buzzer(self) -> bool:
         """True if buzzer is turned on."""
         return self.data["buzzer"] == "on"
 
     @property
+    @setting(
+        name="Led Brightness",
+        icon="mdi:brightness-6",
+        setter_name="set_led_brightness",
+        choices=LedBrightness,
+        entity_category="config",
+    )
     def led_brightness(self) -> Optional[LedBrightness]:
         """LED brightness if available."""
         if self.data["led_b"] is not None:
             return LedBrightness(self.data["led_b"])
         return None
 
     @property
+    @switch(
+        name="Child lock",
```

Comment:
```suggestion
        name="Child Lock",
```
Let's use title case for the names throughout the file, please correct the other instances too.